.travis.yml

```diff
@@ -56,7 +56,6 @@ notifications:
           - http://julia.mit.edu:8000/travis-hook
 before_install:
     - make check-whitespace
-    - JULIA_SYSIMG_BUILD_FLAGS="--output-ji ../usr/lib/julia/sys.ji"
```

Comment:
> bootstrapping is NOT testing the case where you have a ji file but not a dll file

that's exactly what bootstrapping tests.

> which systems without linkers need to continue testing

now that's just a red herring, since this is `make`, we just ran the linker

> changes to tests and CI configuration really do not belong in the same commit as changing the default value here
 
the changes are inter-related (aside from the part where this test has severely bit-rotted)